app/javascript/packages/verify-flow/steps/password-confirm/personal-info-summary.spec.tsx

```diff
@@ -18,4 +18,18 @@ describe('PersonalInfoSummary', () => {
 
     expect(getByText('October 6, 1938')).to.exist();
   });
+
+  it('renders address', () => {
+    const { getByText, rerender } = render(<PersonalInfoSummary pii={DEFAULT_PII} />);
+
+    let address = getByText('1 FAKE RDGREAT FALLS, MT 59010');
+
+    expect([...address.childNodes].filter((node) => node.nodeName === 'BR')).to.have.lengthOf(1);
+
+    rerender(<PersonalInfoSummary pii={{ ...DEFAULT_PII, address2: 'PO BOX 1' }} />);
+
+    address = getByText('1 FAKE RDPO BOX 1GREAT FALLS, MT 59010');
```

Comment:
it's super weird to see the `<br />` get hidden so the lines run together :/

Reply:
Yeah this was a bit clumsy for me to put together, especially since I didn't realize we [previously had an extra space in there](https://github.com/18F/identity-idp/blob/da71f0df731aa05c7859d1be991e3074bfd380cf/app/javascript/packages/verify-flow/steps/password-confirm/personal-info-summary.tsx#L31) as well. After removing that, it makes a _bit_ more sense that the text content is the individual text nodes, joined together with no joiner. 

There are some extra options for matching text (`getByText((text) => ...)`, `getByText('text', { normalizer: ... })`), which could maybe be helpful for this. Or querying the individual slices of text and asserting against its sibling content (e.g. `getByText('1 FAKE RD').nextElementSibling.nodeName === 'BR')`).